### ISSUE TITLE
False positive CVE-2014-1936 Plan 9 rc matching npm rc package

### ISSUE THREAD

**Issue by fouadh** [anchore/grype#934]
**What happened**:
We have [rc](https://github.com/dominictarr/rc) in our dependencies. When scanning our repository we get a false positive on [Plan 9 rc](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=737206) app.

```
rc                               1.2.8                            npm   CVE-2014-1936       High    
```

**What you expected to happen**:
I expect npm rc package not to match CVEs against Plan 9 rc binary.

**How to reproduce it (as minimally and precisely as possible)**:

```
mkdir sandbox
cd sandbox
npm init
npm install rc
grype dir:.
```

**Anything else we need to know?**:

**Environment**:
- Output of `grype version`:
```
Application:          grype
Version:              0.50.1
Syft Version:         v0.56.0
BuildDate:            2022-09-13T18:32:52Z
GitCommit:            403a535321c20565676dc633344e2bf8881cee29
GitDescription:       v0.50.1
Platform:             darwin/amd64
GoVersion:            go1.18.5
Compiler:             gc
Supported DB Schema:  4
```
- OS (e.g: `cat /etc/os-release` or similar):
```
System Software Overview:
      System Version: macOS 12.4 (21F79)
      Kernel Version: Darwin 21.5.0
      Boot Volume: Macintosh HD
```

**Comment by kzantow** [comment]
Thanks for the report, we are currently investigating a few ways to improve these false positives. 👍

**Comment by OfriOuzan** [comment]
We encountered the same issue on the following environment
**What happened**:
In a Vulnerability Scanner Benchmark Research we are conducting, we executed Grype on 20 different containers and found out that Grype has multiple False Positives.
**What you expected to happen**:
We expected Grype not to report on these CVEs.
**How to reproduce it (as minimally and precisely as possible)**:
Install the Docker Images (from the links below) and execute Grype using the following command:
grype <container_name> —-output json > <output_file_path>

- **Output of `grype version`**:
Application:          grype
Version:              0.41.0
Syft Version:         v0.50.0
BuildDate:            2022-07-06T15:20:18Z
GitCommit:            0e0a9d9e7a28592db489499db0294608e5fe69b8
GitDescription:       v0.41.0
Platform:             linux/amd64
GoVersion:            go1.18.3
Compiler:             gc
Supported DB Schema:  4
**Ghost**
 
- **Container Details**:
https://hub.docker.com/layers/library/ghost/5.2.4/images/sha256-42137b9bd1faf4cdea5933279c48a912d010ef614551aeb0e44308600aa3e69f?context=explore

- **OS (e.g: `cat /etc/os-release`)**:
PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
NAME="Debian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
- **CVEs**
**CVE-2014-1936**
Grype wrongly identified CVE-2014-1936 as vulnerable.
The path it identified is:
/var/lib/ghost/versions/5.2.4/node_modules/rc/package.json
The archiver base package version is 1.2.8.
However, according to the debian website, the vulnerability is related to the rc package and the bullseye has a fixed version which is 1.7.4+97.gceb59bb-4.. The versions do not match each other.. So I searched for the rc version in the dpkg and there were no results.. So I run rc and apt suggested to install it using : ‘apt install rc     # version 1.7.4+97.gceb59bb-4..’ which means that rc is not installed.

**Comment by willmurphyscode** [comment]
Hi @fouadh! Thanks for opening the issue and the concise repro script. We believe this is fixed. Creating a new NPM project, installing `rc`, and running `grype dir:. | grep CVE-2014-1936` in it doesn't print any vulnerabilities. 

@OfriOuzan , thanks for the additional reports of the same false positive. I believe they are also resolved:

```
grype --platform linux/amd64 ghost:5.2.4@sha256:42137b9bd1faf4cdea5933279c48a912d010ef614551aeb0e44308600aa3e69f | grep CVE-2014-1936
```
prints no issue. 

Please let us know if you continue experiencing this issue.

**Comment by willmurphyscode** [comment]
Added `changelog-ignore` because this was fixed in `0.60.0` and so shouldn't be included in the current release's release notes.
